www/include/eventLogs/xml/data.php

```diff
@@ -87,13 +88,23 @@
 $inputArguments = array(
     'lang' => FILTER_SANITIZE_STRING,
     'id' => FILTER_SANITIZE_STRING,
-    'num' => FILTER_SANITIZE_NUMBER_INT,
-    'limit' => FILTER_SANITIZE_NUMBER_INT,
+    'num' => [
+        'filter' => FILTER_VALIDATE_INT,
+        'options' => [
+            'default' => 0
+        ]
+    ],
+    'limit' => [
+        'filter' => FILTER_VALIDATE_INT,
+        'options' => [
+            'default' => 30
```

Comment:
could you use this value as default  :
![image](https://user-images.githubusercontent.com/34628915/84034893-3e643100-a99b-11ea-9cf8-29c5b1baa83d.png)

Reply:
_This value is not available in centreon session (optGen array)_

Reply:
check the $optGen['maxViewConfiguration'] ;)

Reply:
![image](https://user-images.githubusercontent.com/34628915/84129623-7d00f680-aa42-11ea-9e95-35f19caf5796.png)